openverse_catalog/dags/providers/provider_api_scripts/cleveland_museum_of_art.py

```diff
@@ -78,64 +79,59 @@ def _get_response(query_param, endpoint=ENDPOINT, retries=RETRIES):
         return response_json, total_images
 
 
+def get_int_value(data, key):
+    value = data.get(key)
+    if value is not None and value != "":
```

Comment:
I _think_ we can just make this a truthy check, since even values like `"0"` will be registered as truthy:
```python
[ins] In [2]: value is not None and value != ""
Out[2]: True

[ins] In [3]: bool(value)
Out[3]: True
```